### PR TITLE
[Bugfix] Cleared Simplex 

### PR DESCRIPTION
**Pull request by kcw-grunt**
- Was causing crashes when return messages were happening.
- Will update in the next cycle with an updated UI
- Added a Analytics:  DID TAP BUY TAB marker

## UI Changes

|Before|After|
|-------|-----|
|Screen Shot 2020-10-15 at 9 48 23 PM|Screen Shot 2020-10-15 at 9 42 43 PM|

### CODE REVIEW

File: loafwallet/BuyTableViewController.swift [hunk 1]
```diff
@@ -8,18 +8,12 @@
...
-    @IBOutlet weak var simplexDetailsLabel: UILabel!
-    @IBOutlet weak var simplexCellContainerView: UIView!
-    
-    @IBOutlet weak var chooseFiatLabel: UILabel!
-    @IBOutlet weak var currencySegmentedControl: UISegmentedControl!
+class BuyTableViewController: UITableViewController {
     
     private var currencyCode: String = "USD"
     
+    private var tempMessage: String = "USD"
```
Comment (mosadialiou) [inline loafwallet/BuyTableViewController.swift:15]: not needed. to be removed!

**Review by mosadialiou — CHANGES_REQUESTED** [review]
Please attach the screenshot

File: loafwallet/BuyTableViewController.swift (hunk 1 above)
Comment (kcw-grunt) [reply loafwallet/BuyTableViewController.swift:15]: yup

**Review by mosadialiou — APPROVED** [review]
LGTM

**Pull request merged**